pylint/config/option_manager_mixin.py

```diff
@@ -271,6 +271,8 @@ def read_config_file(self, config_file=None, verbose=None):
 
         use_config_file = config_file and os.path.exists(config_file)
         if use_config_file:
+            self.current_name = config_file
+            self.set_current_module(self.current_name)
```

Comment:
```suggestion
            self.set_current_module(config_file)
```

This should work as well as `set_current_module` sets the `current_name`.

Reply:
Ha, right, thank you for clarifying !